modules/guides/pages/javascript-udfs/remove-scoped-javascript-function.adoc

```diff
@@ -0,0 +1,74 @@
+= Removing a Namespaced JavaScript Function
```

Comment:
Combine this file with `remove-javascript-udf.adoc`. It's confusing for the user to have two how-to guides covering almost identical procedures.

Reply:
Done